doc/source/dev/conduct/code_of_conduct.md

```diff
@@ -0,0 +1,132 @@
+# SciPy Code of Conduct
+
+SciPy is an engaged and respectful community made up of people
+from all over the world. Your involvement helps us to further our
+mission and to create an open platform that serves a broad range of
+communities, from research and education, to journalism, industry and
+beyond.
+
```

Comment:
A preamble stating the SciPY mission would be helpful here. I think the primary mission is to produce SciPy, that is,  software serving as a basis for the Python scientific stack, and that the aim of the code of conduct is to further that mission by keeping the people working on the project from fighting with each other and creating an unpleasant work environment.